lib/cc/pull_requests.rb

```diff
@@ -6,29 +6,15 @@ def receive_test
   end
 
   def receive_pull_request
-    setup_http
-    state = @payload["state"]
-
-    if %w[pending success failure skipped error].include?(state) && report_status?
-      send("update_status_#{state}")
-    else
-      @response = simple_failure("Unknown state")
-    end
-
-    response
+    receive_request(%w[pending success failure skipped error], :update_status)
```

Comment:
Is there a reason to pass the `call_method` as a symbol instead of a string? I see that we convert it as a string later in the `receive_request` method

Reply:
Honestly, there isn't a strong reason. I just felt like it was more readable, but I'm open to change it if you don't feel the same about it

Reply:
I personally like it as a symbol. I associate symbols with methods better :+1: